src/transformState.js

```diff
@@ -5,7 +5,44 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
-}
+  for (const action of actions) {
+    switch (action.type) {
```

Comment:
You can also destructure the action and get all the needed props of it (type, extraData, keysToRemove)